examples/server-worker/index.ts

```diff
@@ -65,20 +80,20 @@ app.post('/key-shares/generate', async (req: Request, res: Response) => {
   const keyShares = new KeyShares();
   keyShares.update({ operators, publicKey });
 
-  // The nonce of the owner within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool
-  const TEST_OWNER_NONCE = 1;
   // The cluster owner address
   const TEST_OWNER_ADDRESS = '0x81592c3de184a3e2c0dcb5a261bc107bfa91f494';
 
   await keyShares.buildPayload({
     publicKey,
     operators,
     encryptedShares,
-  }, {
+  },{
     ownerAddress: TEST_OWNER_ADDRESS,
-    ownerNonce: TEST_OWNER_NONCE,
+    ownerNonce: nonce,
     privateKey
   });
+  keyShares.payload.readable.encryptedKeys = encryptedShares.map((share: EncryptShare) => share.privateKey);
+  keyShares.payload.readable.publicKeys = encryptedShares.map((share: EncryptShare) => share.publicKey);
```

Comment:
This is also is not the way ot should be used. 
SDK already provided this information, not necessary to build it again